cap/modules/fixtures/schemas/adl-schema.json

```diff
@@ -0,0 +1,72 @@
+{
+	"experiment": "CMS",
+	"use_deposit_as_record": true,
+	"version": "0.0.1",
+	"is_indexed": true,
```

Comment:
change to `is_indexed: false`. we don't need this collection to be searchable on its own, only if used as subschema (so `is_indexed` is not needed)

should be like
```
  "experiment": null,
  "use_deposit_as_record": true,
  "version": "0.0.1",
  "is_indexed": false,
  "allow_all": true,
```